lib/WebServer/LapTriggerWebServer.cpp

```diff
@@ -250,6 +227,106 @@ void LapTriggerWebServer::handleCredentials()
     }
 }
 
+void LapTriggerWebServer::parseWSTextEvent(uint8_t clientId, WStype_t type, uint8_t *payload, size_t length)
+{
+    String cmd;
+    String par;
+    size_t index = 0;
+
+    for (index = 0; index < length; ++index)
+    {
+        char temp = reinterpret_cast<char *>(payload)[index];
```

Comment:
Its better to cast once to a local variable, which is outside the for-loop to avoid performance issues, because of reinterpret_cast().

Reply:
How can I cast the input array to a local variable without going through the for-loop?

Reply:
```char* strPayload = ...```